charmhelpers/contrib/charmsupport/nrpe.py

```diff
@@ -337,10 +337,8 @@ def write(self):
                 "command": nrpecheck.command,
             }
             # If we were passed max_check_attempts, add that to the relation data
-            try:
+            if nrpecheck.max_check_attempts:
```

Comment:
This check is probably correct, but I just need to check that it's not legal for `max_check_attempts` to be `0` (or if that is being disallowed).  More formally, the code should probably read:

```python
if nrpecheck.max_check_attempts is not None:
    ...
```

i.e. explicitly check for the Optional `None` to indicate that it is disabled, prior to 'say' verifying that it is in the correct bounds (if `0` is not a valid number ... which say could be infinite checks??)

Reply:
updated, thanks.